virttest/nfs.py

```diff
@@ -240,7 +240,8 @@ class NFSClient(object):
     def __init__(self, params):
         # Setup SSH connection
         self.ssh_obj = SSHConnection(params)
-        self.ssh_obj.conn_setup()
+        ssh_timeout = int(params.get("ssh_timeout", 10))
+        self.ssh_obj.conn_setup(timeout=ssh_timeout)
```

Comment:
@PandaWei , for migration in tp-libvirt, 'ssh_timeout' will be configured in libvirt/tests/cfg/migration/migrate_vm.cfg, like below:

- virsh.migrate_vm:
    type = migrate_vm
    take_regular_screendumps = no
...
    ssh_timeout = 60